INSTALLATION.md

```diff
@@ -22,20 +22,21 @@ pyenv global 3.10.6
 These two commands set the `python` version to `3.10.6`.
 If you want to go back to your old Python version, run `pyenv global system`.
 
-Once Python 3.10 installed, install Atomkraft with `pip install atomkraft`.
+Once Python is installed, install Atomkraft with `pip install atomkraft`.
 
 That's it! Please verify that the tool is working by issuing `atomkraft` on the command line.
 You should see something like this:
 
 ![Atomkraft CLI](docs/images/cli.png)
 
-## Git
+## Optional dependencies
 
-Atomkraft generates testing projects templates.
-Those templates are obtained using `git`.
-Install `git` following [the installation instructions](https://git-scm.com/downloads).
+Atomkraft is used to test Cosmos-SDK based blockchains.
```

Comment:
Thanks to the English language, `is used to` may mean an activity in part :smile: Please rephrase to something like `is a framework to`.